faraday_middleware.gemspec

```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |spec|
   spec.homepage = 'https://github.com/lostisland/faraday_middleware'
   spec.licenses = ['MIT']
 
-  spec.add_dependency 'faraday', ['>= 0.7.4', '< 1.0']
+  spec.add_dependency 'faraday', '~> 0.16.0'
```

Comment:
faraday 0.17.x and 1.x also use the new namespace.

Technically, 0.17.0 and 0.17.3 i think use the top-level namespace, while 0.17.1 and 0.17.2 provided a shim for the Errors module…